src/Page/Definition.elm

```diff
@@ -329,8 +329,8 @@ expanderDefinitionStyle =
     ]
 
 
-quoteStyle : List Style
-quoteStyle =
+quoteStyles : List Style
```

Comment:
I'm trying to be consistent with naming `xStyles` for List Style as a hint it's a list of them, not one.